SETUP.md

```
@@ -1,5 +1,15 @@
 # SETUP.md
+### Environment Configuration
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)